crates/sui-types/src/temporary_store.rs

```diff
@@ -848,10 +854,9 @@ impl<'backing> TemporaryStore<'backing> {
     pub(crate) fn collect_rebate(&self, gas_charger: &mut GasCharger) {
         for (object_id, kind) in &self.deleted {
             match kind {
-                DeleteKindWithOldVersion::Wrap(version)
-                | DeleteKindWithOldVersion::Normal(version) => {
+                DeleteKindWithOldVersion::Wrap(_) | DeleteKindWithOldVersion::Normal(_) => {
                     // get and track the deleted object `storage_rebate`
-                    let storage_rebate = self.get_input_storage_rebate(object_id, *version);
+                    let storage_rebate = self.get_input_storage_rebate(object_id);
```

Comment:
is it worth continuing to pass version in here so we can add an assert that the version is correct?